src/main/java/main/response/DefaultHeaders.java

```diff
@@ -0,0 +1,17 @@
+package main.response;
+
+import main.date.Date;
+
+public class DefaultHeaders {
+
+    private final Date date;
+
+    public DefaultHeaders(Date date) {
+        this.date = date;
+    }
+
+    public String get() {
+        return date.getDate() + "\n" +
+               "Content-Length: \n";
```

Comment:
I think I would still like to see a proper Content-Length added to each response rather than just giving it a blank Content-length